src/services/emails/companyApplicationApproval.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable max-len */
+
+module.exports = Object.freeze({
+    NEW_COMPANY_APPLICATION_ADMINS: (email, companyName, motivation) => ({
+        subject: `New company application pending review from ${companyName}`,
+        text:
+            `${companyName} <${email}> has requested to participate in NIJobs. Below is their motivation:
+            \n
+            \t${motivation}
+            \n
+            \n           
+            To review and approve or reject this application, log in to NIJobs and go to the Applications Review page (https://ni.fe.up.pt/nijobs/review/applications).
+            \n
+            Sincerely,
+            NIJobs team at NIAEFEUP
+        `,
+        html:
+            `<h3>A company has requested access to NIJobs</h3>
+            <br>
+            <p>${companyName} <${email}> has requested to participate in NIJobs. Below is their motivation:</p>
+            <blockquote><i>${motivation}</i></blockquote>
+            <br>
+            <br>
+            <p>To review and approve or reject this application, log in to NIJobs and go to the <a href="https://ni.fe.up.pt/nijobs/review/applications">Applications Review</a> page.</p>
+            <br>
+            <p>Sincerely,</p>
+            <p>NIJobs team at NIAEFEUP</p>
+        `,
+    }),
+    NEW_COMPANY_APPLICATION_COMPANY: (companyName, applicationId) => ({
+        subject: "Your NIJobs Application",
+        text:
+            `We have successfully received your application!
+            \n
+            We will now review your application, and in case you're approved, you will receive another email with further instructions in order to complete your registration. 
+            \n
+            Your Application ID is ${applicationId} and you registered ${companyName}
+            \n
+            If you did not request this or if you need anything else, don't hesitate to contact us!
+            \n
+            Sincerely,
+            NIJobs team at NIAEFEUP
+        `,
+        html:
+            `<h1>We have successfully received your application!</h1>
+            <p>We will now review your application, and in case you're approved, you will receive another email with further instructions in order to complete your registration.</p>    
+            <p>Your Application ID is ${applicationId} and you registered ${companyName}</p>
+            <br>
+            <p>If you did not request this or if you need anything else, don't hesitate to contact us!</p>
+            <br>
+            <p>Sincerely,</p>
+            <p>NIJobs team at NIAEFEUP</p>
+            
+        `,
+    }),
+    APPROVAL_NOTIFICATION: (companyName) => ({
+        subject: "Your NIJobs Application",
```

Comment:
It is common for the subject on an approval message to already mention that the application was accepted in some way, perhaps something simple such as `Welcome to NIJobs!`? Not sure if this is the best one, but something on these terms sounds better